src/command_modules/azure-cli-interactive/azclishell/app.py

```diff
@@ -679,7 +625,8 @@ def cli_execute(self, cmd):
                     OutputProducer(formatter=formatter).out(result)
                     self.last = result
 
-        except Exception as ex:  # pylint: disable=broad-except
+        except Exception as ex:
+            # pylint: disable=broad-except
```

Comment:
T hese pylint disable comments are clearer on the line that it applies to.
I agree with what Troy mentioned in general but for pylint comments, think on same line is clearer.